### ISSUE TITLE
No startup messages

### ISSUE THREAD

**Issue by jazdw** [infiniteautomation/ma-core-public#975]
Still not working after the change you made yesterday Terry 39cf4766bd23ad83779c7e6c6dd40cd4f1528e67
```
2017-04-02 09:31:01,490 main ERROR Error processing element MangoFrontendAppender ([Appenders: null]): CLASS_NOT_FOUND
2017-04-02 09:31:01,618 main ERROR No appender named frontend was configured
```

**Comment by terrypacker** [comment]
Fixed it up with this commit: c18b9900a3b2b76b3ae2151abaa9c70583d38398

**Comment by jazdw** [comment]
Thanks, working now.
